airbyte-ci/connectors/live-tests/src/live_tests/commons/json_schema_helper.py

```diff
@@ -0,0 +1,265 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+
+
+from functools import reduce
+from typing import Any, Dict, List, Mapping, Optional, Set, Text, Union
+
+import dpath.util
+import pendulum
+from jsonref import JsonRef
+
+
+class CatalogField:
```

Comment:
❓ Was this also taken from CAT?

Did we have any unit tests to accompany?

Reply:
Good call! Copied them over.